### PR TITLE
Include Style in VC, Update submodule, Add Logo

### PR DESCRIPTION
**Pull request by nithyaganeshng**
- Include style in VC
      Added polymorphic serialization initializers for style classes.
      Added/Modified unit tests to verify them.
- Updated the VC SDK to latest code.
- Added Logo to styling, made it serializable and modified unit tests for those.
- Updated GroupRequirement failure to include the list of exceptions in the message.

**Solution:**
Used library A to complete scenario X with quality and style.

**Validation:**
Please include here how it was verified that the PR works as intended.

**Type of change:**
- [x] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/BasicVerifiedIdStyle.kt [hunk 1]
```diff
@@ -17,5 +20,5 @@ class BasicVerifiedIdStyle(
     val description: String,
 
     // Logo that should be displayed on the Verified Id.
-    val logo: Logo? = null
-): VerifiedIdStyle(name)
```
Comment (symorton) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/BasicVerifiedIdStyle.kt:21]: why does name not have to be passed in anymore?

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/VerifiedIdManifestIssuerStyle.kt [hunk 2]
```diff
@@ -10,11 +10,5 @@ package com.microsoft.walletlibrary.requests.styles
...
 data class VerifiedIdManifestIssuerStyle(
     // Name of the requester.
-    override val requester: String,
-
-    // Locale of the requester.
-    val locale: String,
-
-    // Logo of the requester. If a url is present in contract, it is fetched to this.
-    val logo: Logo? = null
+    override val requester: String
```
Comment (symorton) [inline walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/VerifiedIdManifestIssuerStyle.kt]: I have "name" as this property now in iOS. What do you think should be the property name here? I am open to either.

File: walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/handlers/OpenIdRequestHandlerTest.kt [hunk 3]
```diff
@@ -338,7 +337,7 @@ class OpenIdRequestHandlerTest {
         assertThat((actualOpenIdRequest as ManifestIssuanceRequest).verifiedIdStyle).isInstanceOf(
             BasicVerifiedIdStyle::class.java
         )
-        assertThat((actualOpenIdRequest.verifiedIdStyle as BasicVerifiedIdStyle).logo).isNull()
+        assertThat((actualOpenIdRequest.verifiedIdStyle as BasicVerifiedIdStyle).verifiedIdLogo).isNull()
```
Comment (symorton) [inline walletlibrary/src/test/java/com/microsoft/walletlibrary/requests/handlers/OpenIdRequestHandlerTest.kt]: thinking about this a little more in these tests. Can the property name on VerifiedIdStyle still be logo but the class name be VerifiedIdLogo?

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/VerifiedIdManifestIssuerStyle.kt (hunk 2 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/VerifiedIdManifestIssuerStyle.kt]: yeah name is good. I didn't notice the difference. I will change it.

File: walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/BasicVerifiedIdStyle.kt (hunk 1 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/walletlibrary/requests/styles/BasicVerifiedIdStyle.kt:21]: I made VerifiedIdStyle and this class serializable. With that change it cannot have a parameter.

File: walletlibrary/src/test/java/com/microsoft/walletlibrary/mappings/issuance/DisplayContractMappingTest.kt [hunk 4]
```diff
@@ -82,8 +82,8 @@ class DisplayContractMappingTest {
         assertThat(actualVerifiedIdStyle.backgroundColor).isEqualTo(expectedCardBackgroundColor)
         assertThat(actualVerifiedIdStyle.textColor).isEqualTo(expectedCardTextColor)
         assertThat(actualVerifiedIdStyle.logo).isNotNull
-        assertThat(actualVerifiedIdStyle.logo?.uri).isEqualTo(expectedLogoUri)
-        assertThat(actualVerifiedIdStyle.logo?.description).isEqualTo(expectedLogoDescription)
+/*        assertThat(actualVerifiedIdStyle.logo?.url).isEqualTo(expectedLogoUri)
```
Comment (symorton) [inline walletlibrary/src/test/java/com/microsoft/walletlibrary/mappings/issuance/DisplayContractMappingTest.kt]: nit: commented out code

**Review by symorton — DISMISSED** [review]

File: walletlibrary/src/test/java/com/microsoft/walletlibrary/mappings/issuance/DisplayContractMappingTest.kt (hunk 4 above)
Comment (nithyaganeshng) [reply walletlibrary/src/test/java/com/microsoft/walletlibrary/mappings/issuance/DisplayContractMappingTest.kt]: Oh yes, I commented it out to test Uri. I will uncomment it.

**Review by symorton — APPROVED** [review]

**Pull request merged**